metadata_service/proxy/atlas_proxy.py

```diff
@@ -18,23 +19,18 @@
 LOGGER = logging.getLogger(__name__)
 
 
+# noinspection PyMethodMayBeStatic
```

Comment:
just curious: what is this line about?

Reply:
This makes pycharm not to give warning about the method being static.